docs/features/sessions/changing-instance-type.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Run (and sweep) any private or public Github repository.
+---
+import Note from "@site/src/components/Note";
+
+# Change Instance Type of a Session
+
+```text
+grid session change-instance-type [--spot , --on_demand] SESSION_NAME g4dn.xlarge
+```
+
+Changing Sesssion instance type allows you to upgrade or downgrade the compute capacity of the Session while keeping all of your work in progress untouched.
+
+![](/images/sessions/change-instance-type.gif)
+
+
+The equivalent via the CLI command:
+
+![](/images/sessions/change-instance-type.png)
+
+You should PAUSE the session to change the instance type. RESUME the session for the changes to take effect.
```

Comment:
I think we could think while CLI redesign to actually guide user in case of failures. Right now if the session is not paused it would just say that you should pause it first and command would fail. We could actually ask if we should pause it first (y/N) selection, and then do the operation. 
This is purely an enhancement. We should help with either all (or most) or no commands to make the ux flow similar and expected.

Reply:
@adam-grid I agree with that approach